From f4000b08b644c0378f3278ab09d48a7abdd9fd2d Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:50:16 -0400
Subject: [fs] nfsd: Remove 'inline' designation for free_client()

Message-id: <1403560314-32525-75-git-send-email-bfields@redhat.com>
Patchwork-id: 84559
O-Subject: [Patch RHEL7 074/172] NFSd: Remove 'inline' designation for free_client()
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Trond Myklebust <trond.myklebust@primarydata.com>

It is large, it is used in more than one place, and it is not performance
critical. Let gcc figure out whether it should be inlined...

Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 4dd86e150f63a6c360783f163a979ec563e6d570
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4state.c b/fs/nfsd/nfs4state.c
index 32b699b..841495a 100644
--- a/fs/nfsd/nfs4state.c
+++ b/fs/nfsd/nfs4state.c
@@ -1093,7 +1093,7 @@ static struct nfs4_client *alloc_client(struct xdr_netobj name)
  return clp;
 }
 
-static inline void
+static void
 free_client(struct nfs4_client *clp)
 {
  struct nfsd_net __maybe_unused *nn = net_generic(clp->net, nfsd_net_id);
-- 
1.7.1